dangerfile.js

```diff
@@ -1,17 +1,37 @@
 import { fail, danger } from 'danger';
-import { flatten, intersection, isEmpty, includes } from 'lodash';
+import { flatten, intersection, isEmpty } from 'lodash';
 
 const pkg = require('./package.json'); // eslint-disable-line import/newline-after-import
 const prLogConfig = pkg['pr-log'];
 
+const Versions = {
+  PATCH: 'PATCH',
+  MINOR: 'MINOR',
+  MAJOR: 'MAJOR',
+};
+
+const branchVersion = Versions.PATCH;
```

Comment:
This variable should be configured per branch. Maybe this should be a field in `package.json`, I'm not sure